src/main/java/org/alfresco/rest/api/model/Node.java

```diff
@@ -99,6 +99,7 @@
     //optional SearchEntry (only ever returned from a search)
     protected SearchEntry search = null;
     protected String location;
+    private Boolean favorite;
```

Comment:
Please rename favorite to isFavorite so we're consistent with the naming convention for boolean in Node.